src/Notepads/Controls/TextEditor/TextEditorCore.DateTime.cs

```diff
@@ -6,11 +6,28 @@
 
     public partial class TextEditorCore
     {
+        private bool _hasAddedLogEntry = false;
+
         private void InsertDateTimeString()
         {
             var dateStr = DateTime.Now.ToString(CultureInfo.CurrentCulture);
             Document.Selection.SetText(TextSetOptions.None, dateStr);
             Document.Selection.StartPosition = Document.Selection.EndPosition;
         }
+
+        public void TryInsertNewLogEntry()
```

Comment:
Add a summary to this method and paste some links describing this feature since only few people knows about it.

Reply:
Done.